torch/nn/modules/conv.py

```diff
@@ -37,14 +37,14 @@ class _ConvNd(Module):
     def __init__(self,
                  in_channels: int,
                  out_channels: int,
-                 kernel_size: _size_1_t,
-                 stride: _size_1_t,
-                 padding: _size_1_t,
-                 dilation: _size_1_t,
+                 kernel_size: Tuple[int, ...],
+                 stride: Tuple[int, ...],
+                 padding: Tuple[int, ...],
+                 dilation: Tuple[int, ...],
```

Comment:
Again, why this change is needed? `padding=2` seems like a valid syntax, isn't it?

Reply:
Yes, child classes can take ints, but they are always converted to tuple prior to initialization of the parent construction, e.g. Conv1d calls `kernel_size = _single(kernel_size)`